src/pyj/read_book/paged_mode.pyj

```diff
@@ -40,14 +39,9 @@ def has_start_text(elem):
     return False
 
 def handle_rtl_body(body_style):
-    # Make the body and root nodes have direction ltr so that column layout
-    # works as expected
     if body_style.direction is "rtl":
-        for node in document.body.childNodes:
-            if node.nodeType is Node.ELEMENT_NODE and window.getComputedStyle(node).direction is "rtl":
-                node.style.setProperty("direction", "rtl")
-        document.body.style.direction = "ltr"
-        document.documentElement.style.direction = 'ltr'
```

Comment:
I needed to disable this, since it makes columns in RTL books display in the wrong order, left to right instead of right to left.